app/components/DataTable/components/SvgViewer/index.jsx

```diff
@@ -11,6 +11,8 @@ import {
 import { UncontrolledReactSVGPanZoom } from 'react-svg-pan-zoom';
 import InlineSVG from 'svg-inline-react';
 import AutoSizer from 'react-virtualized/dist/es/AutoSizer';
+// not ideal, but no errors are thrown and no render loops occur
+// eslint-disable-next-line import/no-cycle
 import DataTable from '../..';
```

Comment:
very scared of this

Reply:
ya, this is a bit sketchy. @Anna-log7  might be a good thing to make a github/jira issue/ticket to fix this in a future PR unless it's something you can easily fix now

Reply:
I'm open to suggestions on how to fix this, but I thought about it for a bit and the only thing I could come up with was using a new AgGridReact table which isn't DataTable in here

Reply:
This issue also affects the `GeneViewer` component to `DataTable`

Reply:
I think just making a ticket detailing the problem is sufficient for this PR since its more about the linting, you can work on fixing the circular imports now that they are highlighted in a subsequent one. 

I know I had a similar problem for graphkb client at one point and I fixed it by splitting the bulk of the component out and making 2 top-level components that used it. one imported the other

Reply:
I just ended up fixing it as I described above